### DISCUSSION TITLE
Train models

### DISCUSSION THREAD

**Discussion by mhzawadi** [nextcloud/recognize#355] [Q&A] [answered]
Hello 👋🏼 

I have now got this up and running on my 24.0.5, its now found a load of faces and thats amazing!!
But I want to change who an image is, how do I go about that?

**Comment by marcelklehr** [comment, marked as answer]
I recommend upgrading to Nextcloud v25 which has people management functionality in the new photos app that will allow you to manage and merge faces.

**Reply by mhzawadi** [reply]
I did think that, do you know when v25 will land?
I've seen v25.0.1 RC in github

**Reply by marcelklehr** [reply]
v25.0.0 is out already since last week. [v25.0.1 will land on 2022-11-03](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule)

**Reply by mhzawadi** [reply]
I have been keeping an eye on the nextcloud site, but its missing the links for v25
<img width="461" alt="image" src="https://user-images.githubusercontent.com/11944327/198661909-de998453-7aa6-4105-a743-5a109409e813.png">

But the `Maintenance and Release Schedule` page is gold

**Reply by tcpipuk** [reply]
The current version definitely has "People" features to let you merge/remove faces, but I can't see a way to move individual photos around.

I have a "face" that's been tagged on ~20 photos when it's actually about 10 different people (and a cat!) and all I can really do is remove photos or remove the whole face... if it affects training, is there some way to split them up into difference faces?
